projects/scion/toolkit.internal/widgets/src/tabbar/tabbar.component.scss

```diff
@@ -0,0 +1,47 @@
+:host {
+  display: flex;
+  flex-direction: column;
+
+  > sci-viewport.tabbar {
+    flex: none;
+    height: 2em;
+
+    ul.tabs {
+      display: flex;
+      flex-flow: row nowrap;
+      list-style: none;
+      margin: 0;
+      padding: 0;
+
+      > li.tab, > li.filler {
+        flex: none;
+        border-bottom: 3px solid var(--sci-color-P300);
+        padding: 0 1em 0 1em;
+        white-space: nowrap;
+
+        &:hover {
```

Comment:
Also exclude the filler, as following:
```&:hover:not(.filler)```
or combine it with the subsequent style, as following:
```
&:hover:not(.filler) {
  cursor: pointer;

  &:not(.selected) {
    border-bottom-color: var(--sci-color-P400);
    color: var(--sci-color-accent);
  }
}
```

Reply:
done